dashboard/modules/snapshot/tests/test_component_activities.py

```diff
@@ -0,0 +1,175 @@
+import os
```

Comment:
I assume this file is just moving existing code?

Reply:
Yea the diff rendered poorly but this is the subset of relevant tests from existing `test_snapshot.py`